ReactiveCocoa.podspec

```diff
@@ -24,7 +24,7 @@ Pod::Spec.new do |s|
   s.watchos.source_files = "ReactiveCocoa/WatchKit/*.{swift}"
   s.module_name = 'ReactiveCocoa'
 
-  s.dependency 'ReactiveSwift', '~> 4.0'
+  s.dependency 'ReactiveSwift', '~> 5.0'
 
   s.pod_target_xcconfig = { "OTHER_SWIFT_FLAGS[config=Release]" => "$(inherited) -suppress-warnings" }
   s.swift_version = '4.1.2'
```

Comment:
This should be `4.2`.

Reply:
Should we merge one of the Swift 4.2 branches first then?

Reply:
Ah okay this is not required, I'm sorry for confusing you!

(Both Swift 4.2 compiler and Swift 5 compiler support Swift 4 mode, so this is still valid)